pilot/pkg/xds/debug.go

```diff
@@ -330,15 +330,26 @@ func (s *DiscoveryServer) distributedVersions(w http.ResponseWriter, req *http.R
 			con.proxy.RLock()
 
 			if con.proxy != nil && (proxyNamespace == "" || proxyNamespace == con.proxy.ConfigNamespace) {
+				// if there is no value from last nonce,
+				// "-1" should be set for identification of no last nonce
+				sclusterType := s.StatusReporter.QueryLastNonce(con.ConID, v3.ClusterType)
```

Comment:
This variable name seems deceptive.  This appears to be holding the most recent cluster nonce.

Reply:
@therealmitchconnors yes, I can change the name of variable, how about `sclusterTypeNonce`, `slistenerTypeNonce` and `srouteTypeNonce` ?

Reply:
how about just clusterNonce, listenerNonce, etc?  I think Type is redundant, and I'm not sure what the 's' prefix indicated...

Reply:
it's okay